stock_picking_force_assign/models/stock_picking.py

```diff
@@ -24,6 +24,12 @@ def action_force_assign_pickings(self):
         )))
         return self.action_assign()
 
+    def _force_assign_allow_partial(self):
+        """Indicate if partial forced assignments should be allowed"""
+        return self.env["ir.config_parameter"].sudo().get_param(
+            "stock_picking_force_assign.allow_partial"
+        ).strip().lower() in ("1", "true", "yes")
```

Comment:
You could use [`str2bool`](https://github.com/odoo/odoo/blob/6f8b65c5a0c65d2c1a49f152423e69bfa6b3896d/odoo/tools/misc.py#L454)?

Reply:
Yes, I can, thanks! Refactored accordingly. I was a bit surprised that using `tools.str2bool` gave me the one from `tools/convert.py` instead of this one, so I selected it explicitely.